src/web_crawler/twir_crawler.rs

```diff
@@ -238,7 +254,7 @@ impl TwirCrawler {
             error!("writing the content file failed: {e}");
             CrawlerError::Fetch
         })?;
-
+        
         Ok(())
     }
-}
+}
```

Comment:
Please run cargo fmt --all after you make the changes from the comments above so it will format your code and commit the formated code as well